tests/tests_pytorch/strategies/test_ddp_hydra_support.py

```diff
@@ -0,0 +1,168 @@
+import logging
```

Comment:
If we could move the file to `.../strategies/launchers/test_subprocess_script.py` that would be helpful, thanks!